e2e/tests/query.feature

```diff
@@ -0,0 +1,391 @@
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account 2BBCA5DC9792C72AC3A2616910C4AAAA 0"
+	Then the user should be able to see standard output containing "Incorrect address length"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account, wrong height
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account {{address}} 5600000" against validator 0
+	Then the user should be able to see standard output containing "wanted to load target 5600000 but only found up to"
+	And the pocket client should have exited without error
+
+  Scenario: To query an non-existing Account
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account c7b6a62385f8999c0cb63ad9cb464ee730c597b2 0"
+	Then the user should be able to see standard output containing "null"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account, incomplete command
+	Given the user has a pocket client
+	When the user runs the command "query account"
+	Then the user should be able to see standard error containing "Usage:"
+	And the pocket client should have exited with error
+
+  Scenario: To query an existing account txs
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account-txs c7b6a62385f8999c0cb63ad9cb464ee730c597b1 0 1 0 0" against validator 0
+	Then the user should be able to see standard output containing "page_count"
+	And the user should be able to see standard output containing "total_txs"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account txs, wrong address
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query account-txs c7b6a62385f8999c0cb63ad9cb464ee730c597bw 0 1 0 0" against validator 0
+	Then the user should be able to see standard output containing "encoding/hex: invalid byte:"
+	And the pocket client should have exited without error
+
+  Scenario: To query an existing account txs, incomplete command
+	Given the user has a pocket client
+	When the user runs the command "query account-txs"
+	Then the user should be able to see standard error containing "Usage:"
+	And the pocket client should have exited with error
+
+  Scenario: To query current acl
+	Given the user is running the network "single_node_network"
+	And the user has a pocket client
+	When the user runs the command "query acl"
+	Then the user should be able to see standard output containing "/v1/query/acl"
+	And the user should be able to see standard output containing "gov/non_map_acl"
+	And the pocket client should have exited without error
```

Comment:
NIT: We can also take advantage of indentation to improve readability. For example, the above scenario could be indented as follows:
```
Scenario: To query current acl
	Given the user is running the network "single_node_network"
	   And the user has a pocket client
	  When the user runs the command "query acl"
	  Then the user should be able to see standard output containing "/v1/query/acl"
	   And the user should be able to see standard output containing "gov/non_map_acl"
	   And the pocket client should have exited without error
```

Reply:
The style of indentation I am using is consistent with the style used by cucumber.io in the [Gherkin language reference ](https://cucumber.io/docs/gherkin/reference/)